src/components/Firestore/DocumentEditor/index.tsx

```diff
@@ -85,8 +95,10 @@ const DocumentEditor: React.FC<{
 const RootField: React.FC<{
   onChange?: (value: FirestoreMap) => void;
   areRootKeysMutable: boolean;
-}> = ({ onChange, areRootKeysMutable }) => {
+  limitToOneField: boolean;
```

Comment:
/** Some docs on exactly what this means*/

Reply:
Done